internal/util/plugins/makefile.go

```diff
@@ -97,5 +99,10 @@ bundle: kustomize
 # Build the bundle image.
 bundle-build:
 	docker build -f bundle.Dockerfile -t $(BUNDLE_IMG) .
+`
+	// Makefile fragments to add to the base Makefile.
+	makefilePackagemanifests = `
+packagemanifests: kustomize
+	$(KUSTOMIZE) build config/manifests | operator-sdk generate packagemanifests --version $(VERSION)
 `
```

Comment:
the target was missing be scaffolded for all types. So it was added as described in the docs. See: https://master.sdk.operatorframework.io/docs/olm-integration/quickstart-package-manifests/#creating-package-manifests

Reply:
The doc is meant for users who want to _continue_ using package manifests format, not for new OLM users; the docs even say this. That's why there's no `packagemanifests` Makefile target by default.

Reply:
ok. So we should not scaffold this by default.